scene/audio/audio_stream_player.cpp

```diff
@@ -162,7 +162,9 @@ void AudioStreamPlayer::_notification(int p_what) {
 	}
 
 	if (p_what == NOTIFICATION_UNPAUSED) {
-		set_stream_paused(false);
+		if (can_process()) {
+			set_stream_paused(false);
+		}
```

Comment:
I don't think this check should be necessary. If a node receives this notification, it should be able to process.

Reply:
Removed ;)

Reply:
I've created a new method. Some classes call `can_process()` when reacting to pause/unpause notification, so I'm calling the new method after propagating pause owner, as it can change the result.